planner/orienteering_router.py

```diff
@@ -27,38 +28,38 @@ def __init__(self, gmaps_api_key: str, conn):
         self.conn = conn
         self.cur = conn.cursor()
 
-    def get_parks_from_gmaps(self, location: Tuple[float, float],
-                             radius: float) -> List[GmapsResult]:
+    def get_pois_from_gmaps(self, location: Tuple[float, float],
+                             radius: float, type_list: List[str]) -> List[GmapsResult]:
         """
         Get POIs from Google Maps.
 
         Get POIs from Google Maps Places API Nearby Search. It only uses a
         single API request, and only returns up to 20 places.
         :param location: (lat, lon) pair.
         :param radius: Search radius in meters.
+        :param types: List of the type of POIS being fetched from gmaps
+            (parks, landmarks, coffee shops, etc.)
         :return: List of results.
         """
-        res = self.gmapsclient.places_nearby(
-            location=(34.145265, -118.130473), # lat, lon
-            radius=5000, # meters
-            type='park'
-        )
-
+        GoogleHelper = GoogleUtils.GoogleHelper() 
+        res = GoogleHelper.get_pois({'lat': location[0], 'lng': location[1]}, radius=radius, type_list=['park'])
+        #GoogleUtils.save_to_json(res, file_name='output.txt')
         output = []
-        for result in res['results']:
+
+        for place in res.places:
             try:
                 output.append(self.GmapsResult(
-                    name=result['name'],
-                    latlon=(result['geometry']['location']['lat'],
-                            result['geometry']['location']['lng']),
-                    rating=result['rating']
+                    name=place.name,
+                    latlon=(place.geo_location['lat'], place.geo_location['lon']),
+                    rating=place.rating
                 ))
             except KeyError:
                 # Skip POIs that are missing one of the fields
                 pass
 
```

Comment:
This function wasn't returning any POIs, because each POI threw a KeyError in this block. I think the key for the longitude is supposed to be 'lng', and the try/except block might not be needed anymore.

Reply:
Oops - fixed